x/logic/interpreter/fs/fs.go

```diff
@@ -0,0 +1,99 @@
+package fs
```

Comment:
I would like the `fs` package to be outside the `interpreter` package because it can be fully decorrelated and independant I guess.

Reply:
Yes, totally agree, but I kept it into the logic module I think.

Reply:
All right!